Helpers.js

```diff
@@ -0,0 +1,223 @@
+}
+
+
+/**
+ * Removes triggers by handler function.
+ *
+ * Given a function name, all triggers execute said function when fired will be
+ * removed from the AppScript project.
+ *
+ * @param {string} functionName The name of the function run by the trigger.
+ */
+function deleteTriggers(functionName) {
+  for (const trigger of ScriptApp.getProjectTriggers() ?? []) {
+    if (trigger.getHandlerFunction() === functionName) {
+      ScriptApp.deleteTrigger(trigger);
+    }
+  }
+}
+
+/**
+ * Remove Batch Triggers
+ */
+function removeBatchTriggers(){
+  deleteTriggers('runBatch');
+  deleteTriggers('runBatchPSI');
+  deleteTriggers('runBatchPSIWithScreenshots');
+  deleteTriggers('runBatchCrUXHistory');
+  deleteTriggers('runBatchCrUX');
+}
+
+
+/**
+ * Get a simple value for tests
+ */
+function quickTestForValue(){
+  Logger.log(SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Config").getRange("B26").getValue())
+}
+
+
+
+/**
+ * Resets the Spreadsheet to default values
+ */
+function resetURLsToDefault(){
+  SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Config").getRange("D2:I").deleteCells(SpreadsheetApp.Dimension.ROWS)
+  SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Config").getRange("D2:H7").setValues(SpreadsheetApp.getActiveSpreadsheet().getSheetByName("web.dev Websites").getRange("A2:E7").getValues());
+}
+
+/**
+ * Delete all the data in Results tab
+ */
+function deleteData(){
+  // // Results
+  // let last_row = SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Results").getLastRow();
+  // SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Results").deleteRows(3,last_row+1-3)
+  // // Screenshots
+  // last_row = SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Screenshots").getLastRow();
+  // SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Screenshots").deleteRows(3,last_row+1-3)
+
+  let sheets = ["Results", "Screenshots", "Debug", "Accessibility", "Green Domains (GWF)", "Sustainability"]
+  for(let i = 0; i < sheets.length; i++){
+    let sheet_name = sheets[i]
+    let last_row = SpreadsheetApp.getActiveSpreadsheet().getSheetByName(sheet_name).getLastRow();
+    if(last_row >= 5){
+      SpreadsheetApp.getActiveSpreadsheet().getSheetByName(sheet_name).deleteRows(3,last_row+1-3)
+    }
+  }
+
+}
+
+/**
+ * Helper to understand the time it takes the Spreadsheet to flush
+ */
+function timePerformance() {
+  var startTime = new Date();
+  // Trigger a change in the sheet, for example, write to a cell
+  SpreadsheetApp.getActiveSpreadsheet().getSheetByName('Results').getRange('G2').setValue(Math.random());
+  SpreadsheetApp.flush(); // Apply all pending changes immediately
+  var endTime = new Date();
+  
+  var executionTime = endTime - startTime;
+  Logger.log("Execution Time: " + executionTime + " ms");
+  toast("Execution Time: " + executionTime + " ms");
+}
+
+
+
+/**
+ * Show an alert that we will call a 3P if the API is Green Domains
+ */
+function showAlertCallGWF() {
```

Comment:
The naming and the logic here is confusing. I know there are schools of thought on avoiding negative booleans, but the functions need to be named appropriately so that the expected return values are clear. Looking at where this is called in PSI_Tracker.js, I was also confused and thought there was a logic error at first.

Reply:
done with `showConfirmationModalCallGWF`